storage/innobase/row/row0ins.cc

```diff
@@ -2819,6 +2831,8 @@ row_ins_clust_index_entry_low(
 				goto avoid_bulk;
 			}
 
+			row_ins_write_autoinc(index, auto_inc, mode,
+					      &mtr);
```

Comment:
Do we really need this write in the bulk insert code path here, or could it be done as part of `BtrBulk::pageCommit(…, false)` or `BtrBulk::finish()` later?